tests/stability/runtimeclass_workloads/pod-deployment.yaml

```diff
@@ -0,0 +1,24 @@
+# Copyright (c) 2024 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: deploymenttest
+spec:
+  selector:
+    matchLabels:
+      purpose: pod-test
+  replicas: 2
```

Comment:
@GabyCT , should the replicas field have value 8 - the default value from kubernetes_soak_test.sh?

Reply:
@danmihai1 this will at the beginning launch 2 pods or replicas but as the test continues then we scale up the number of pods to 8

Reply:
Ah, I missed the kubectl scale. Thanks!